### PR TITLE
chore: react-native-nitro-totp

### PR DESCRIPTION
**Pull request by 4cc3ssX**
# 📝 Why & how

Added `react-native-nitro-totp` module and updated `unmaintained`, `alternatives` field to `react-native-totp-utils`.

# ✅ Checklist

- [x] Added library to **`react-native-libraries.json`**
- [x] Updated library in **`react-native-libraries.json`**

### CODE REVIEW

File: react-native-libraries.json [hunk 1]
```diff
@@ -9647,7 +9647,10 @@
     "githubUrl": "https://github.com/4cc3ssX/react-native-totp-utils",
     "examples": ["https://github.com/4cc3ssX/react-native-totp-utils/tree/main/example"],
     "ios": true,
-    "android": true
+    "android": true,
+    "newArchitecture": true,
```
Comment (Simek) [inline react-native-libraries.json:9651]:
```suggestion
    "newArchitecture": true,
```

Is this correct?

**Review by Simek — COMMENTED** [review]
Hello @4cc3ssX, thanks for the updates, and adding new entry to the directory! 👌 

Just have one question, about the new architecture status for the deprecated package.

File: react-native-libraries.json (hunk 1 above)
Comment (4cc3ssX) [reply react-native-libraries.json:9651]: Yes it's correct. Compatible with both new and old architectures while alternative one only supports new architecture.

Comment (Simek) [reply react-native-libraries.json:9651]: Thanks for confirming! Let's rebase the PR so we can ship it.

Comment (4cc3ssX) [reply react-native-libraries.json]: I've to pull to resolve conflicts and discarded my changes. Creating new PR for now.

**Pull request closed without merging**